src/ray/core_worker/core_worker.cc

```diff
@@ -1458,9 +1458,12 @@ Status CoreWorker::CreateActor(const RayFunction &function,
         "Async actor is currently not supported for the local mode");
   }
   const auto next_task_index = worker_context_.GetNextTaskIndex();
+  // NOTE(swang): Include the current time in the hash for the actor ID so that
```

Comment:
Does the task index become unused above then? Should we mix it with the timestamp?

Reply:
The task index still gets populated in the task spec, but I'm not really sure where that gets used.

Sure, I'll try to mix them.